src/main/java/sirius/biz/process/ErrorContext.java

```diff
@@ -258,10 +283,11 @@ public void handleInContext(UnaryOperator<String> failureDescription, String lab
         try {
             return Optional.ofNullable(producer.create());
         } catch (HandledException exception) {
-            logException(exception);
+            logException(exception, ProcessLogType.ERROR);
         } catch (Exception exception) {
             String message = exception.getMessage() + " (" + exception.getClass().getName() + ")";
-            logException(Exceptions.handle().to(Log.BACKGROUND).error(exception).withDirectMessage(message).handle());
+            logException(Exceptions.handle().to(Log.BACKGROUND).error(exception).withDirectMessage(message).handle(),
+                         ProcessLogType.ERROR);
```

Comment:
Use new `logError()`?
```suggestion
            logError(Exceptions.handle().to(Log.BACKGROUND).error(exception).withDirectMessage(message).handle());
```

Reply:
logException is correct, since we receive a HandledException for logging. Error and Warning is a concept for the ProcessLog, so I changed the API methods to logExceptionAsError and logExceptionAsWarning since it better represents what they do.